judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/validation/ValidationController.java

```diff
@@ -63,5 +64,28 @@ public ValidationController(
         return toDtos(collect, serviceName, serviceVersion);
     }
 
+    @GetMapping(value = "/environment-compatibility-report/{serviceName}:{serviceVersion:.+}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "Get validation report for contract between given service and given environment", nickname = "Validate against environment")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Success", response = String.class, responseContainer = "list"),
+        @ApiResponse(code = 500, message = "Failure"),
+        @ApiResponse(code = 404, message = "Service not found"),})
```

Comment:
unnecessary comma, format it like this for better readability:

```java
@ApiResponses(value = {
        @ApiResponse(code = 200, message = "Success", response = String.class, responseContainer = "list"),
        @ApiResponse(code = 500, message = "Failure"),
        @ApiResponse(code = 404, message = "Service not found")
})
```

also you should consider to make usage of repeatable annotations here so it could look like this:

```java
@GetMapping(value = "/environment-compatibility-report/{serviceName}:{serviceVersion:.+}", produces = MediaType.APPLICATION_JSON_VALUE)
@ApiOperation(value = "Get validation report for contract between given service and given environment", nickname = "Validate against environment")
@ApiResponse(code = 200, message = "Success", response = String.class, responseContainer = "list")
@ApiResponse(code = 500, message = "Failure")
@ApiResponse(code = 404, message = "Service not found")
public List<ContractValidationReportDto> validateAgainstEnvironments() { 
    // something ...
}
```